tests/www/test_views.py

```diff
@@ -639,10 +638,10 @@ def test_view_uses_existing_dagbag(self, endpoint):
         self.check_content_in_response('example_bash_operator', resp)
 
     @parameterized.expand([
-        ("hello\nworld", r'\"conf\":{\"abc\":\"hello\\nworld\"}}'),
-        ("hello'world", r'\"conf\":{\"abc\":\"hello\\u0027world\"}}'),
-        ("<script>", r'\"conf\":{\"abc\":\"\\u003cscript\\u003e\"}}'),
-        ("\"", r'\"conf\":{\"abc\":\"\\\"\"}}'),
+        ("hello\nworld", r'\"conf\":{\"abc\":\"hello\\nworld\"}'),
```

Comment:
Out of curiosity - where this change came from?

Reply:
Extra params now appear in the output, so the closing `}` is now a `,`.